Tables/obs4MIPs_Aday.json

```diff
@@ -1,6 +1,6 @@
 {
     "Header":{
-        "Conventions":"CF-1.7 CMIP-6.0",
+        "Conventions":"CF-1.7 ODS-2.0",
         "approx_interval":"1.00000",
         "cmor_version":"3.2",
         "data_specs_version":"01.00.12",
```

Comment:
@durack1 We need to complete versions for an intital set of both ODS CVs and modified-CMIP6-to-obs4MIPs CMOR tables so that data providers can start to include it.